packages/handshake-api/package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "handshake-api",
+  "version": "1.0.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "punycode": {
+      "version": "2.1.1",
+      "resolved": "https://registry.npmjs.org/punycode/-/punycode-2.1.1.tgz",
+      "integrity": "sha512-XRsRjdf+j5ml+y/6GKHPZbrF/8p2Yga0JPtdqTIY2Xe5ohJPD9saDJJLPvp9+NSBprVvevdXZybnj2cv8OEd0A=="
+    }
+  }
+}
```

Comment:
use `yarn workspace handshake-api add punycode` to add `punycode` dependency

you can delete this file too, we're not supporting npm since it doesn't support workspaces